decentraland/unity-renderer
# feat: [SUB-BRANCH] Implement channel list of players and counter

## What does this PR change?
Fix #2735 

In order to know who are you talking with within channels, players need a way to know the number of players connected to the same channel.

The players will be listed in a new section, accessible by a button in the channel conversation view.
The channels will display the number of connected players next to their name in the conversations list view.

![image](https://user-images.githubusercontent.com/64659061/187423980-a077542b-c063-468c-bde0-421767d8977f.png)

**Figma design:** https://www.figma.com/file/SxxomcIZI1zyytMZmFJAvS/Social-Improvements?node-id=3303%3A39380
**Data Flow:** https://hackmd.io/sMgwddkUR_SDZvWqLlFfeg?view#Ability-to-get-the-list-of-users-joined-to-a-channel

## How to test the changes?
1. Go to: https://play.decentraland.org/?renderer-branch=feat/implement-channel-list-of-players-and-counter&kernel-branch=feat/add-friends-lazy-load
2. Join any channel.
3. Expand the new members section.
4. Notice a list of users are loaded.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
